app/services/doc_auth/error_generator.rb

```diff
@@ -339,12 +337,12 @@ def generate_doc_auth_errors(response_info)
       unknown_error_handler.handle(response_info).to_h
     end
 
-    def self.general_error(_liveness_enabled)
+    def self.general_error
       Errors::GENERAL_ERROR
     end
 
-    def self.wrapped_general_error(liveness_enabled)
-      { general: [ErrorGenerator.general_error(liveness_enabled)], hints: true }
+    def self.wrapped_general_error
+      { general: [ErrorGenerator.general_error], hints: true }
     end
```

Comment:
I'm not too familiar with how this all is abstracted, but does it make sense to have these methods at all anymore, vs. inlining `Errors::GENERAL_ERROR` and `{ general: [Errors::GENERAL_ERROR], hints: true }` ?

Reply:
I can see what you mean. The argument was made at standup that this value is used in many places, and it would be much simpler to only update it here if it changes (or if we decide later on to show a different general error if liveness is enabled).

But the person who made the argument did say they don't feel that strongly about it, so if given that you think the inline solution is better, I can make that change.

Reply:
I think a stronger case could at least be made for getting rid of the `general_error` method, since it's only ever used internal to this class, and it would bring consistency in places where we're otherwise not using abstracted references to specific errors.

Example:

https://github.com/18F/identity-idp/blob/6c2bebc895b53ac377206e1c5a8dbc9638064c04/app/services/doc_auth/error_generator.rb#L153-L160

Reply:
Makes sense to me, thanks for explaining! Updated in 531d9696400894eff7b8c5d4f4efc1c9595c8af0. I did leave `wrapped_general_error` but I was conflicted about it. It's only used in one place so it does seem like it could be removed. On the other hand though, even though it's just one line, it seems like a lot of logic that belongs to the `ErrorGenerator` than the `TrueIDResponse`, so I opted to leave it. I could be convinced otherwise though.